### DISCUSSION TITLE
Thanks

### DISCUSSION THREAD

**Discussion by PreviST** [Stirling-Tools/Stirling-PDF#463] [General]
I'd been looking for an application like this for a long time to avoid using online services. Thank you very much, it's great.

**Comment by trebestie** [comment]
I agree with @PreviST, awesome job @Frooodle 
If there was a box to decrypt *.p7m signed pdf, it would be almost perfect for me.

**Comment by NODRT** [comment]
I agree! This app is great. I just installed it in Docker. All I had to do was rearrange scrambled pdf pages, which seems so simple but it's not available on any other pdf app.

Thanks to the developers!
